modules/imgproc/perf/perf_warp.cpp

```diff
@@ -2,7 +2,7 @@
 // It is subject to the license terms in the LICENSE file found in the top-level directory
 // of this distribution and at http://opencv.org/license.html.
 #include "perf_precomp.hpp"
-
+#include "opencv2/ts/ts_perf.hpp"
```

Comment:
Please remove this unnecessary include, it should be handled by `perf_precomp.hpp` (via `opencv2/ts.hpp`).